src/main/resources/META-INF/maven/site.vm

```diff
@@ -161,82 +166,86 @@
 #*      *##end
 ##
         </div>
       </div>
-    </div>
+    </header>
 #**##end
 ##
 #**##if ( $sideBarEnabled )
     <div class="container-fluid">
 #**##else
     <div class="container">
 #**##end
-      <div id="banner">
-        <div class="pull-left">##
-#*      *##if( $decoration.bannerLeft )
-#*        *##banner( $decoration.bannerLeft "bannerLeft" )
-#*      *##else
-#*      *### MSKINS-32
-          <h1>##
-#*        *##if( $project.name && $project.name != '' )
-#*          *#$project.name
+      <header>
+        <div id="banner">
+          <div class="pull-left">##
+#*        *##if( $decoration.bannerLeft )
+#*          *##banner( $decoration.bannerLeft "bannerLeft" )
 #*        *##else
-#*          *#$project.artifactId
+#*        *### MSKINS-32
+            <h1>##
+#*          *##if( $project.name && $project.name != '' )
+#*            *#$project.name
+#*          *##else
+#*            *#$project.artifactId
+#*          *##end
+#*        *#</h1>##
 #*        *##end
-#*      *#</h1>##
-#*      *##end
-#*    *#</div>
-        <div class="pull-right">#banner( $decoration.bannerRight "bannerRight" )</div>
-        <div class="clear"><hr/></div>
-      </div>
+#*      *#</div>
+          <div class="pull-right">#banner( $decoration.bannerRight "bannerRight" )</div>
+          <div class="clear"><hr/></div>
+        </div>
 
-      <div id="breadcrumbs">
-        <ul class="breadcrumb">
-#*      *##publishDate( "left" $decoration.publishDate $decoration.version )
-#*      *##publishDate( "right" $decoration.publishDate $decoration.version )
+        <div id="breadcrumbs">
+          <ul class="breadcrumb">
+#*        *##publishDate( "left" $decoration.publishDate $decoration.version )
+#*        *##publishDate( "right" $decoration.publishDate $decoration.version )
 ##
-#*      *### MSITE-44
-#*      *##if ( !$topBarEnabled
-              && $decoration.body.links
-              && $decoration.body.links.size() > 0 )
-#*        *##links( $decoration.body.links )
-#*      *##end
-        </ul>
-      </div>
+#*        *### MSITE-44
+#*        *##if ( !$topBarEnabled
+                && $decoration.body.links
+                && $decoration.body.links.size() > 0 )
+#*          *##links( $decoration.body.links )
+#*        *##end
+          </ul>
+        </div>
+      </header>
 ##
 #*    *##if ( $sideBarEnabled )
 #*      *##set ( $leftColumnClass = $decoration.getCustomValue( 'fluidoSkin.leftColumnClass', 'span2' ) )
 ##
       <div class="row-fluid">
-        <div id="leftColumn" class="$leftColumnClass">
-          <div class="well sidebar-nav">
+        <header id="leftColumn" class="$leftColumnClass">
+          <nav class="well sidebar-nav">
 #*      *##publishDate( "navigation-top" $decoration.publishDate $decoration.version )
 #*      *##mainMenu( $decoration.body.menus )
 #*      *##publishDate( "navigation-bottom" $decoration.publishDate $decoration.version )
+          </nav>
+          <div class="well sidebar-nav">
```

Comment:
Why is this not a `nav` too?

Reply:
I understand <nav> is supposed to surround navigation elements, but specifically exclude non -navigation links. The links excluded are things like the google search, facebook like button, etc, none of which are related to navigation.